app/assets/stylesheets/modules/base_styles.scss

```diff
@@ -1,3 +1,15 @@
 body {
   overflow-x: hidden;
 }
+
+a {
```

Comment:
I'll take this for a spin tomorrow; I remember trying this and ended up with borders where they didn't belong (like in the global footer, and other oddities)

Reply:
Oh, heh, good point, and it looks like we are getting some of those oddities in SearchWorks currently. 😅